content/english/ml-machine-learning/02-simple-linear-regression.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Simple Linear Regression"
+description: "Make computers learn to predict outcomes."
+prereq: "Python"
+icon: ""
+draft: false
+weight: 2
+---
+
+# What is simple linear regression?
+Simple linear regression aims to find a correlation between two variables. One dependent and one independent. With regression in general, we want to know:
+
+1. Is there a **relationship** between the variables we have?
+
+    As an example, you can determine the relationship between income and spending, experience and salary, or humidity and temperature. But there is NO relation between a student's height and their exam scores.
+
+2. Can we **forecast / predict** values with this?
+
+    With regression, we can train the model and find out if we can predict values with certainty. Can we use what we know about the relationship to predict new values?
+    
+    Example: What will be the temperature tomorrow? How much will my bakery sell this year compared to last year? How much will my salary be if I have 5 years of experience?
+
+# Variable roles
+In simple linear regression, variables can take one of 2 roles.
+
+1. **Dependent Variable**
+
+    The variable whose value we want to predict or forecast. We call it dependent because its value depends on something else. We call this variable **y**.
+
+2. **Independent Variables**
+
+    The variable that explains the behavior of the dependent variable. We call this variable **x**.
+
+# The Simple Linear Equation Model
+
+When we use simple linear regression we call it **linear** because, well... the model for it is an equation that represents a straight line in a 2D plane.
+
+What is the equation of a straight line?
+{{% expand "**Click to show answer**" %}}
+This equation may seem familiar to you. If it is, it's the general equation for a straight line.
+
+<h1>
+\[
+    y = ax + b
```

Comment:
Adding a graph here to give it a visual aspect of what the equation looks like. We also talked about potentially adding a link to a graphing site so they can play around with it optionally